underscore.js

```diff
@@ -1149,7 +1149,7 @@
     if (a !== a) return b !== b;
     // Exhaust primitive checks
     var type = typeof a;
-    if (type !== 'function' && type !== 'object' && typeof b !== 'object') return false;
+    if (type !== 'function' && type !== 'object' && typeof b != 'object') return false;
```

Comment:
@megawac yes it definitely is, however changing it to `type != 'function'` causes the travis build to fail as it doesn't pass the linter.